.gitignore

```diff
@@ -1,2 +1,3 @@
+npm-debug.log
```

Comment:
This isn't an expected file. I would consider simply removing it from local environment instead of adding it in .gitignore.